ui/src/SurveyDetails.js

```diff
@@ -294,29 +295,25 @@ class SurveyDetails extends Component {
 
     return (
       <div style={{ padding: 20 }}>
+        <Link to="/">← Back to home</Link>
         <Typography variant="h4" color="inherit" style={{ marginBottom: 20 }}>
-          Survey: {this.props.surveyName}
+          Survey Details
```

Comment:
Would be helpful to keep the survey name somewhere on this screen

Reply:
I know... but let's put on another ticket because it was only a 1 day spike